test/scripts/valhalla_build_config.py

```diff
@@ -0,0 +1 @@
+/home/nilsnolde/dev/cpp/valhalla/scripts/valhalla_build_config
```

Comment:
this is the symlink

Reply:
of course this wont work on CI. i think you need to link it like this with relative paths

```bash
# clean up bad link
cd /home/nilsnolde/dev/cpp/valhalla/test/scripts
rm valhalla_build_config.py
# add new relative link
ln -s ../../scripts/valhalla_build_config valhalla_build_config.py
```

Reply:
😂 didn’t even realize that..